src/pages/AnimeSub/main.ts

```diff
@@ -0,0 +1,101 @@
+import { pageInterface } from '../pageInterface';
+
+export const AnimeSub: pageInterface = {
+  name: 'AnimeSub',
+  domain: 'https://animesub.lt',
+  languages: ['Lithuanian'],
+  type: 'anime',
+  isSyncPage() {
+    if (window.location.href.split('/')[6] !== '') return true;
+    return false;
+  },
```

Comment:
please add a `isOverviewPage` too. This can help to ignore unknown pages sometimes